lib/sidekiq/batch/callback.rb

```diff
@@ -5,10 +5,9 @@ class Worker
         include Sidekiq::Worker
 
         def perform(clazz, event, opts, bid)
-          return unless %w(success complete).include?(event)
```

Comment:
Actually I was little wrong with the previous comment. Can we have that line above just to be safe?